enabler/test/de/schildbach/pte/live/CostaRicaProviderLiveTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2010-2015 the original author or authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.schildbach.pte.live;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import de.schildbach.pte.CostaRicaProvider;
+import de.schildbach.pte.dto.Location;
+import de.schildbach.pte.dto.LocationType;
+import de.schildbach.pte.dto.NearbyLocationsResult;
+import de.schildbach.pte.dto.QueryDeparturesResult;
+import de.schildbach.pte.dto.SuggestLocationsResult;
+
+/**
+ * @author Jaime Gutiérrez Alfaro
+ */
+public class CostaRicaProviderLiveTest extends AbstractNavitiaProviderLiveTest {
+    public CostaRicaProviderLiveTest() {
+        super(new CostaRicaProvider(secretProperty("navitia.authorization")));
```

Comment:
If your server requires authorization (does it?), it is probably different from the one of the official navitia server, right? So in that case this should probably use a different secret variable here.